bin/Levels/l7978.lvl

```diff
@@ -28,175 +28,179 @@
-function CancelDrowning()
-	Game.StopTimer("tdrown");adrown=0
-	for i=1,5,1 do Game.SetEntityPos("sdrown"..i,0,1000); end
-	Game.SetGravity(0,-9.81)
-	min_distance=4 --(fireflies come back!)
-end
-function StartDrowning()
-	ox=x;oy=y;x,y=Game.GetPlayerPosition()
-	Game.StartTimer("tdrown",300,5)
-	adrown=1; drownmeter=5
-end
-
---CURSE OF THE TWELWE STONEMEN
-zcurse={};tcurse={};cursemsg=0
-function zcurse.OnEnter()
-	if cursemsg==0 then Game.Message("Oh no! This area is Cursed!"); cursemsg=1 end
-	Game.StartTimer("tcurse",400,0)
-	Game.PlayMusic("batcave")
-end
-function zcurse.OnLeave()
-	Game.StopTimer("tcurse")
-	Game.SetGravity(0,-9.81)
-	Game.PlayMusic("madeirastew")
-end
-function tcurse.Tick(n)
-	if n==1 then Game.SetTimerDelay("tcurse",50) end
-	Game.SetGravity(0,-9.81-(n*5))
-end
-
---FISHIES
-function FishInit()
-	for i=0,3,1 do		
-		dur=math.random(1000,3000)
-		Game.SetDynamicBlockTranslation("fish"..i,math.random(-1.5,1.5),math.random(-0.6,-0.1),dur,0,0)
-	end
-end
-zfish={};afish=0
-function zfish.OnEnter()
-	if afish==0 then
-		afish=1
-		for i=0,3,1 do		
-			dur=math.random(50,100)
-			Game.SetDynamicBlockTranslation("fish"..i,-(2-i)*math.random(0.9,1.4),math.random(-1.2,-0.8),dur,0,dur/2)
-		end
-	end
-end 
-
---HAT
-zhat={};ahat=0
-function zhat.OnEnter()
-	if ahat==0 then
-		ahat=1
-		Game.Message("That must be her hat! I wonder what happened...")
-	end
-end
-
---SPIKES!
-zspike1={};zspike2={};zspike3={}
-function zspike1.OnLeave() Game.SetDynamicBlockTranslation("bspike1",3.5,0,100,0,100) end
-function zspike2.OnLeave() Game.SetDynamicBlockTranslation("bspike2",-4,0,100,0,100) end
-function zspike3.OnLeave() Game.SetDynamicBlockTranslation("bspike3",0,4,100,0,100) end
-
---TICK TACK TOCK TUCK and so on...
-function Tick()
-	--drowning
-	if adrown==1 then --render bubbles
-		ox=x;oy=y;x,y=Game.GetPlayerPosition()
-		min_distance=0 --(fireflies escape!)
-		if tdm&gt;2 then
-			tdm=0
-			x2,y2=Game.GetPlayerPosition();x2=x2+1;xy2=y2+2
-			for i=drownmeter,1,-1 do
-			  Game.SetEntityPos("sdrown"..i,x2,y2)
-			  y2=y2+0.3
-			end
-		end
-		tdm=tdm+1
-		if tw&gt;20 then --save in rpl size!
-			tw=0
-			--water has resistance
-			Game.SetGravity(-(x-ox)*200,(9.81*-1)-(y-oy)*200)
-		end
-	end
-	tw=tw+1
-	return true
-end
-
---WINDMILL
-zmill={};last_t=Game.GetTime()
-function zmill.OnEnter() 
-	if (last_t+2)&lt;Game.GetTime() then
-		last_t=Game.GetTime()
-		Game.SetDynamicBlockSelfRotation("bmill", -1000, 0, 15)
-		Game.SetDynamicBlockSelfRotation("bmill", 500, 15, 250)
-		Game.PlaySound("Squeek")
-	end
-end
-
---MUD
-zmud={};tmud={};zmuddeath={}
-function zmud.OnEnter() Game.AddForceToPlayer(0,-1000,0,30,0); Game.StartTimer("tmud",10,0) end
-function zmud.OnLeave() Game.StopTimer("tmud") end
-function tmud.Tick() Game.AddForceToPlayer(0,-1000,0,30,0) end
-function zmuddeath.OnEnter() Game.KillPlayer() end
-
+function OnLoad()
```

Comment:
I didn't actually change all this lines, this was just the result of running dos2unix on the file.

The real change was the creation of the rand_float(l, u) function, and the replacement of all the math.random(float, float) with this function.